cmd/juju/caas/add.go

```diff
@@ -202,7 +213,24 @@ func (c *AddCAASCommand) Init(args []string) (err error) {
 		if c.contextName != "" {
 			return errors.New("do not specify context name when adding a GKE cluster")
 		}
+		if c.k8sCluster == nil {
+			c.k8sCluster = newGKECluster()
+		}
+		if err := c.k8sCluster.ensureExecutable(); err != nil {
+			return errors.Trace(err)
+		}
 	} else {
```

Comment:
put else if on one line

Reply:
I'm not sure I follow this, this else includes checks for --aks and for when neither aks/gke is passed. Perhaps I need to be clearer here?